xc7/make/project_xray.cmake

```diff
@@ -99,7 +124,7 @@ function(PROJECT_XRAY_TILE)
     --output-model ${CMAKE_CURRENT_BINARY_DIR}/${TILE}.model.xml
     ${FUSED_SITES_ARGS}
     DEPENDS
-    ${TILE_IMPORT}
+    ${TILE_TYPE_IMPORT}
```

Comment:
Typo?  Should still be TILE_IMPORT?

Reply:
Yes indeed, I swapped again the two of them. I think I need to find a better name to distinguish between the tiles. What about:
- `prjxray_physical_tile_import.py` instead of `prjxray_tile_type_import.py` and - 
- `prjxray_pb_type_import.py` instead of `prjxray_tile_import.py`?